src/HDInsight/HDInsight/ManagementCommands/GetAzureHDInsightPropertiesCommand.cs

```diff
@@ -20,9 +20,8 @@
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
-    [GenericBreakingChange("Get-AzHDInsightProperties alias will be removed in an upcoming breaking change release", "2.0.0")]
```

Comment:
Ditto.

Reply:
> Ditto.

Hi @msJinLei , the reason is that the breaking change notice said the alias will be removed at version 2.0.0.  Now the stable version is 2.0.1. So maybe someone forget to remove the alias and breaking change notice.  So I remove the alias and breaking change notice.

Reply:
> Ditto.

have added it back.